### PR TITLE
fix #2

### LINKED ISSUE #2: truncate options not works correctly
first use case

``` js
related(['micromatch'], {truncate: false}, function (err, res) {
  console.log(res);
  //=> '[micromatch](https://github.com/jonschlinkert/micromatch): '
});
```

and second use case when `truncate:true` truncates to first word, same also if more than one package given in array

``` js
related(['micromatch'], {truncate: false}, function (err, res) {
  console.log(res);
  //=> '[micromatch](https://github.com/jonschlinkert/micromatch): Glob… [more](https://github.com/jonschlinkert/micromatch)'
});
```

### PR DESCRIPTION
**Pull request by tunnckoCore**
fix #2 plus new test cases, I guess 100% coverage

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = function (options) {
     }
 
     opts = extend({}, options, opts);
-    var linkify = opts.linkify || toLink;
+    var linkify = typeof opts.linkify === 'function' ? opts.linkify : toLink;
     if (opts && opts.silent !== true) message();
```
Comment (jonschlinkert) [inline index.js:31]: good call

**Comment by jonschlinkert** [comment]
actually, this isn't related to your pr, what is generated when there is no description? or if the description is one word or something? 

thanks! I'll merge later today, I need these changes!

**Comment by tunnckoCore** [comment]
> what is generated when there is no description

it will be null string so it will return nothing to description. something like 

```
- [assemble](http://assemble.io):
- [micromatch](https://github.com/jonschlinkert/micromatch):
```

but yea, good point!

I'll commit fix. Have you some package in mind that have empty/one-word description?

File: index.js (hunk 1 above)
Comment (tunnckoCore) [inline index.js:31]: yea, I always do type checks like that, lol

**Comment by tunnckoCore** [comment]
so yea, handled :)

``` js
/* empty description */
truncate('', 'http://localhost');
//=> ''
truncate('', 'http://localhost', false);
//=> ''
truncate('', 'http://localhost', 5);
//=> ''

/* one word description */
truncate('foo', 'http://localhost');
//=> ': foo'
truncate('foo', 'http://localhost', false);
//=> ': foo'
truncate('foo', 'http://localhost', 5);
//=> ': foo'

/* otherwise */
truncate('foo bar baz faz qux cat dog', 'http://localhost', false);
//=> ': foo bar baz faz qux cat dog'
truncate('foo bar baz faz qux cat dog', 'http://localhost');
//=> ': foo bar baz faz… [more](http://localhost)'
truncate('foo bar baz faz qux cat dog', 'http://localhost', 2);
//=> ': foo bar… [more](http://localhost)'
```

**Comment by tunnckoCore** [comment]
I love myself when first write the use cases than think the logic, so damn easy! lol

commit comin'

**Comment by tunnckoCore** [comment]
@jonschlinkert and that's it, green light. we only need package with empty description and one-word description to match 100%, lol

**Comment by tunnckoCore** [comment]
https://github.com/tunnckoCore/npm-related waits this merge, it is CLI on top of that

**Comment by jonschlinkert** [comment]
manually merging... accidentally merged in work I already did before I remembered this.

**Comment by jonschlinkert** [comment]
resolved by https://github.com/helpers/helper-related/commit/2294139f50a5c672da29be2eccfc57a4e7c9eadf. Thanks!

**Pull request closed without merging**